samples/extensions/dynamic_state3_multisample_rasterization/README.adoc

```diff
@@ -0,0 +1,61 @@
+////
+- Copyright (c) 2023, Mobica Limited
+-
+- SPDX-License-Identifier: Apache-2.0
+-
+- Licensed under the Apache License, Version 2.0 the "License";
+- you may not use this file except in compliance with the License.
+- You may obtain a copy of the License at
+-
+-     http://www.apache.org/licenses/LICENSE-2.0
+-
+- Unless required by applicable law or agreed to in writing, software
+- distributed under the License is distributed on an "AS IS" BASIS,
+- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+- See the License for the specific language governing permissions and
+- limitations under the License.
+-
+////
+
+// Extended dynamic_state3: Rasterization samples
+
+
+== Overview
+
+This sample demonstrates one of the functionalities of VK_EXT_extended_dynamic_state3 related to rasterization samples.
+The extension can be used to dynamically change sampling without any need to restart the application.
+image:image/image.png[]
+
+== Enabling the extension
+
+To be able to use this extension in Vulkan API:
+1) At least Vulkan API version 1.1 must be supported.
+2) Set of `VkPhysicalDeviceExtendedDynamicStateFeaturesEXT`, `VkPhysicalDeviceExtendedDynamicState2FeaturesEXT`, `VkPhysicalDeviceExtendedDynamicState3FeaturesEXT`
+must be added to the feature chain of `VkPhysicalDeviceProperties2` and `VkPhysicalDeviceExtendedDynamicState3PropertiesEXT` must be added to `VkPhysicalDeviceProperties2`.
```

Comment:
As far as I can tell, are you only using `VkPhysicalDeviceExtendedDynamicState3FeaturesEXT` in this sample. And it is added to the `VkDeviceCreateInfo`-chain (and the `VkPhysicalDeviceFeatures2`-chain), not the `VkPhysicalDeviceProperties2`-chain.
`VkPhysicalDeviceExtendedDynamicState3PropertiesEXT` is not used in this sample.